install/tools/istio_vm_common.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+
+# Helper functions for extending the cluster with external VMs.
+
```

Comment:
Cluster -> mesh 
Helper functions for extending istio mesh outside kubernetes

Reply:
Done